### PR TITLE
Fixed grammar, typos, replaced "span" with Span<T>

### PR DESCRIPTION
**Pull request by jzabroski**
Offering up some improvements to the original copy writing/docs for Span<T>.

### CODE REVIEW

**Comment by wesamco** [comment]
this feature is Awesome
at least we can help with typos
`perfromance => performance`

File: docs/Span.md [hunk 1]
```diff
@@ -74,18 +74,18 @@ In addition, we will provide a read-only version of Span\<T\>. The ReadOnlySpan\
...
 
-The stack-only nature of spans (see more on this below), allows pooled memory to be safely returned to the pool after the transformations pipeline complete, and allows the pipeline to pass only the relevant slice of the buffer to each transformation routine/component. In other words, Span\<T\> aids in lifetime management of pooled buffers, so critical to perfromance of today's servers.
+The stack-only nature of Span<\T\> (see more on this below), allows pooled memory to be safely returned to the pool after the transformations pipeline complete, and allows the pipeline to pass only the relevant slice of the buffer to each transformation routine/component. In other words, Span\<T\> aids in lifetime management of pooled buffers, so critical to perfromance of today's servers.
```
Comment (svick) [inline docs/Span.md]: You've got the backslashes wrong here, it should be `Span\<T\>`, not `Span<\T\>`.

File: docs/Span.md [hunk 2]
```diff
@@ -163,12 +163,12 @@ To support the scenarios described above, Span\<T\> must meet the following requ
...
 2.	Performance characteristics on par with arrays
-3.	Be memory safe
+3.	Be memory-safe
 
 #Design/Representation
 We will provide two different implementations of Span\<T\>: 
-- Fast Span (avaliable on runtimes with special support for spans)
-- Slow Span (avaliable on all current .NET runtimes, even existing ones, e.g. .NET 4.5) 
+- Fast Span<\T\> (avaliable on runtimes with special support for spans)
+- Slow Span<\T\> (avaliable on all current .NET runtimes, even existing ones, e.g. .NET 4.5)
```
Comment (svick) [inline docs/Span.md]: The same here.

File: docs/Span.md [hunk 3]
```diff
@@ -209,47 +209,47 @@ For most structs, tearing is at most a correctness bug and can be dealt with by
...
 
 ##Conversions
 Span\<T\> will support reinterpret cast conversions to Span\<byte\>. It will also support unsafe casts between arbitrary primitive types. The reason for this limitation is that some processors don’t support efficient unaligned memory access.
 
 ##Platform Support Plans
-We want to ship Span\<T\> as a nugget fat package available for .NET Standard 1.1 and above. Runtimes that support by-ref fields and returns will get the fast two filed span. Other runtimes will get the slower three-field span. 
+We want to ship Span\<T\> as a NuGet fat package available for .NET Standard 1.1 and above. Runtimes that support by-ref fields and returns will get the fast two "ref field" Span<\T\>. Other runtimes will get the slower three field Span\<T\>.
```
Comment (svick) [inline docs/Span.md]: And here.

**Comment by jzabroski** [comment]
I also fixed typos in the code samples. I missed a few the other PR caught.
I also standardized stack-only spelled as such.

Also... CLR Type Loader -> CLR Type Activator, since it is an instance of
the type, as opposed to the blueprint the type provides, that we are making
tearing-safe.

By the way, I dislike the phrase stack-only as I have heard many times how
that is too low level a detail to write directly into the CLR Ecma
standard. Compare to ThreadLocal <T>.

**Comment by jzabroski** [comment]
Thanks. I will fix and resubmit. It is my first time doing Github Markdown.

**Comment by KrzysztofCwalina** [comment]
I merged these manually as there were many conflicts. But great thanks for a very detailed review!

**Comment by jzabroski** [comment]
Thank you Petr and Krzystof.  It is wonderful documentation. Cheers.

**Pull request closed without merging**